TTS/tts/datasets/formatters.py

```diff
@@ -334,21 +334,22 @@ def vctk(root_path, meta_files=None, wavs_path="wav48_silence_trimmed", mic="mic
     return items
 
 
-def vctk_old(root_path, meta_files=None, wavs_path="wav48"):
+def vctk_old(root_path, meta_files=None, wavs_path="wav48", ignored_speakers=None):
     """homepages.inf.ed.ac.uk/jyamagis/release/VCTK-Corpus.tar.gz"""
-    test_speakers = meta_files
     items = []
     meta_files = glob(f"{os.path.join(root_path,'txt')}/**/*.txt", recursive=True)
     for meta_file in meta_files:
         _, speaker_id, txt_file = os.path.relpath(meta_file, root_path).split(os.sep)
         file_id = txt_file.split(".")[0]
-        if isinstance(test_speakers, list):  # if is list ignore this speakers ids
-            if speaker_id in test_speakers:
+        # ignore speakers
+        if isinstance(ignored_speakers, list):
+            if speaker_id in ignored_speakers:
                 continue
         with open(meta_file, "r", encoding="utf-8") as file_text:
             text = file_text.readlines()[0]
         wav_file = os.path.join(root_path, wavs_path, speaker_id, file_id + ".wav")
-        items.append([text, wav_file, "VCTK_old_" + speaker_id])
```

Comment:
Why do you need to change this?

Reply:
Because without this we will receive the error:  `vctk_old() got an unexpected keyword argument 'ignored_speakers`. And I'm still using the old VCTK version (The same one used in the YourTTS and SC-GlowTTS papers).

Reply:
VCTK_old is old. So if you don't emphasize, how would we know what dataset has been used for the experiments?

You better fix the naming in your old runs than change the code.

Reply:
The VCTK OLD and VCTK new have the same speakers and as far I know they are the same dataset when we are considering the "mic1" (default). If someone wants to load mic 1 (using the old formatter) and mic2 (using the new formatter) it will create new speakers for each formatter. What no make sense (because they are the same speaker in the end just different samples). If we have the config.json we can easily check if it was trained with VCTK_OLD or VCTK new formatter. So I guess that the better is keep VCTK_x for both.